### ISSUE TITLE
First Problem in HW4

### ISSUE THREAD

**Issue by JakeTheFlare** [Montana-Media-Arts/441-web-technologies#11]
Does anyone get the first problem in the HW4 assignment?  I'm just completely confused right now.

**Comment by JonSwallow** [comment]
function funcDeclaration() {
    return 'A function declaration';
}

var funcExpression = function () {
    return 'A function expression';
}

the waters were muddied labeling it definition. Instead of declaration.

**Comment by ellenkuehl** [comment]
Jon's function looks correct- that's what I have for the first problem as well!

**Comment by marchendrickson** [comment]
It asked us to log it into the console so instead of return shouldn't you use the `console.log(the result)`

**Comment by JonSwallow** [comment]
@marchendrickson  You are right I used console.log in mine. What I posted earlier was the best example I could find as far as the difference between a declaration and expression. Good call I should have been clearer as well. Cheers!!

**Comment by michaelmusick** [comment]
You all are on the right track!

**Comment by JakeTheFlare** [comment]
Thank you so much everyone.
